miniml/compiler/compile.scm

```diff
@@ -1476,6 +1513,32 @@
     (bytecode-emit-labref lab2)
   ))
 
+(define (compile-arg-name a i)
+  (let ((p (get-arg-pat a)))
+    (cond
+     ((equal? (car p) 'PVar)
+      (car (cdr p)))
+     (else
+      (string-append "arg" (number->string i))))))
```

Comment:
Could you use a string such as "arg#" to avoid any possible conflict with an argument with that name?

Reply:
Indeed, thanks. (I renamed the `function` argument, currently just `#` for the same reason, into `arg#function` for consistency.)

Reply:
(I rebased and pushed.)